### ISSUE TITLE
RCS1192 is incorrectly reported

### ISSUE THREAD

**Issue by nvmkpk** [JosefPihrt/Roslynator#617]
**Product and Version Used**: 2.2

**Code**:
```
@$"{DateTime.UtcNow:yyyy\/MM\/dd\/HH\/mm}"
```
RCS1192 is reported in the above line. But if that refactoring is applied then it removes the '@' symbol and that leads to compiler error because `'\/'` is not a valid escape sequence.

**Comment by nvmkpk** [comment]
If I replace this code with `DateTime.UtcNow.ToString(@"yyyy\/MM\/dd\/HH\/mm")` then it is not reported.
